moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -111,6 +114,48 @@ export const useStyles = makeStyles(theme => ({
   ...layerSelectStyles,
 }));
 
+const removeFeatureFromCollection = (selectedFeature, featureCollection) => {
```

Comment:
Nothing really new in these three functions—i just isolated the logic for updating the feature collection because we need to call it conditionally based on if/how we address fragmented features